main.js

```diff
@@ -1,20 +1,83 @@
 import repos from "/data/repos.js";
 
 export default function buildPage() {
-    const root = document.querySelector("div#root");
-    root.innerText = "Hello!";
+    // Creating table
+    let root = document.querySelector("div#root");
+    createTable(root, 4, repos.length);
+    function createTable(parent, cols, rows) {
+        let table = document.createElement("table");
+        let headerRow = document.createElement("tr");
+        let headers = ["Name", "Description", "Link", "Updated At"];
+
+        for (let h = 0; h < cols; h++) {
+            let th = document.createElement("th");
+            th.innerText = headers[h];
+            headerRow.appendChild(th);
+        }
+        table.appendChild(headerRow);
+        
+        for (let i = 0; i < rows; i++) {
```

Comment:
if you follow my suggestion above, and refactor function as `createTable(data) `, you then will be able to use `Array.map()` method here, and map all repo objects into `tr` objects.
then you can find the way how to append all of that as child into table either one by one or all at once.